modules/internal/ChapelTaskData.chpl

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2004-2017 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// ChapelTaskData.chpl
+//
+module ChapelTaskData {
+
+  use ChapelStandard;
+
+  // Chapel task-local data format:
+  // up to 16 bytes of wide pointer for _remoteEndCountType
+  // 1 byte for serial_state
+  private const chpl_offset_endCount = 0:size_t;
```

Comment:
Spacing here looks a little weird. I think either align them or (my personal preference) just drop to 1 space after `const`